membership/tests.py

```diff
@@ -1130,13 +1130,19 @@ class LoginRequiredTest(TestCase):
     fixtures = ['membership_fees.json', 'test_user.json']
 
     def setUp(self):
+        membership_o = create_dummy_member('N', type='P')
+        membership_o.type = 'P'
+        membership_o.save()
+        user = User.objects.get(id=1)
+        membership_o.preapprove(user)
+        membership_o.approve(user)
         self.urls = ['/membership/memberships/new/',
                      '/membership/memberships/preapproved/',
                      '/membership/memberships/preapproved-plain/',
                      '/membership/memberships/approved/',
                      '/membership/memberships/approved-emails/',
                      '/membership/memberships/deleted/',
-                     '/membership/memberships/convert_to_an_organization/1/'
```

Comment:
I have zero idea how this has worked before with combined url /membership/memberships/convert_to_an_organization/1//membership/memberships/